lets-talk.fifthtry.site/.is-local

```diff
@@ -1 +1 @@
-We are developing this package from this folder.
+We are developing this package from this folder.
```

Comment:
@pypdeveloper this red icon means the file does not end with newline. All files should end with new line. Looks like your editor automatically strips newline at the end of file. Fix your editor setting and undo these changes.